theano/gpuarray/extra_ops.py

```diff
@@ -451,24 +453,23 @@ def c_support_code_struct(self, node, nodename):
         return super(GpuCumsum, self).c_support_code_struct(node, nodename) + code
 
 
+@register_opt('fast_compile')
 @op_lifter([CumsumOp])
-def use_gpu_cumsumop(node, ctx_name):
-    if node.inputs[0].dtype == 'float32':
-        axis = node.op.axis
-        x = node.inputs[0]
-
+@register_opt2([CumsumOp], 'fast_compile')
+def use_gpu_cumsumop(op, ctx_name, inputs, outputs):
```

Comment:
Can you rename it to local_gpua_cumsum() to have the same pattern name as what we have elsewhere?